src/pybind/mgr/orchestrator/module.py

```diff
@@ -982,7 +982,7 @@ def _service_rm(self,
         """Remove a service"""
         if service_name in ['mon', 'mgr'] and not force:
             raise OrchestratorError('The mon and mgr services cannot be removed')
-        completion = self.remove_service(service_name)
+        completion = self.remove_service(service_name, force)
```

Comment:
Changing this "remove_service" call in the orchestrator module to have this extra parameter requires changing the "remove_service" declaration in interface to also have that parameter.
https://github.com/ceph/ceph/blob/af36f892196a2c967d7cbdc209f8f36ec63f7fc2/src/pybind/mgr/orchestrator/_interface.py#L479

Reply:
verified by mypy! 

```
orchestrator/__init__.py:3: note: In module imported here:
orchestrator/module.py: note: In member "_service_rm" of class "OrchestratorCli":
orchestrator/module.py:985: error: Too many arguments for "remove_service" of "Orchestrator"

Reply:
ouch  ... i forget to add one file ....